lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,316 @@
+        mongo_comment3 = self.db.contents.find_one({'author_username':self.sql_users[2].username})
+
+        #ensure that the starting mongo usernames match the starting sql usernames
+        self.assertTrue(mongo_user1['username'] == self.sql_users[0].username)
+        self.assertTrue(mongo_comment1['author_username'] == self.sql_users[0].username)
+
+        self.assertTrue(mongo_user2['username'] == self.sql_users[1].username)
+        self.assertTrue(mongo_comment2['author_username'] == self.sql_users[1].username)
+
+        self.assertTrue(mongo_user3['username'] == self.sql_users[2].username)
+        self.assertTrue(mongo_comment3['author_username'] == self.sql_users[2].username)
+
+
+
+        #perform the rename only for the second user
+        args = [self.sql_users[1].username, TestRenameUser.NEW_USERNAME]
+        self.rename_cmd.handle(*args)
+
+
+        #requery sql and mongo users
+        self.sql_users[0] = User.objects.get(id=self.sql_users[0].id)
+        self.sql_users[1] = User.objects.get(id=self.sql_users[1].id)
+        self.sql_users[2] = User.objects.get(id=self.sql_users[2].id)
+
+        mongo_user1 = self.db.users.find_one({'_id':mongo_user1['_id']})
+        mongo_user2 = self.db.users.find_one({'_id':mongo_user2['_id']})
+        mongo_user3 = self.db.users.find_one({'_id':mongo_user3['_id']})
+
+        mongo_comment1 = self.db.contents.find_one({'_id':mongo_comment1['_id']})
+        mongo_comment2 = self.db.contents.find_one({'_id':mongo_comment2['_id']})
+        mongo_comment3 = self.db.contents.find_one({'_id':mongo_comment3['_id']})
+
+        #check that the rename was successful for only the second user
+        self.assertNotEqual(self.sql_users[0].username, TestRenameUser.NEW_USERNAME)
+        self.assertNotEqual(mongo_user1['username'], TestRenameUser.NEW_USERNAME)
+        self.assertNotEqual(mongo_comment1['author_username'], TestRenameUser.NEW_USERNAME)
+
+        self.assertEqual(self.sql_users[1].username, TestRenameUser.NEW_USERNAME)
+        self.assertEqual(mongo_user2['username'], TestRenameUser.NEW_USERNAME)
+        self.assertEqual(mongo_comment2['author_username'], TestRenameUser.NEW_USERNAME)
+
+        self.assertNotEqual(self.sql_users[2].username, TestRenameUser.NEW_USERNAME)
+        self.assertNotEqual(mongo_user3['username'], TestRenameUser.NEW_USERNAME)
+        self.assertNotEqual(mongo_comment3['author_username'], TestRenameUser.NEW_USERNAME)
+
+
+
+
+    @classmethod
+    def tearDownClass(cls):
+        #remove test mongo users
+        for test_user_id in cls.mongo_user_ids:
+            cls.db.users.remove({"_id":test_user_id})
+        #remove test mongo comments
+        for test_comment_id in cls.mongo_comment_ids:
+            cls.db.contents.remove({"_id":test_comment_id})
+        #sql users are flushed before each test because this class inherits from django.test.TestCase
+
```

Comment:
I think you actually have to delete the users created by `setUpClass`, based on the broken tests at https://jenkins.class.stanford.edu:8080/job/Stanford_Online_Unit_Tests_auto_pr/375/consoleFull